kombu/transport/SQS.py

```diff
@@ -160,15 +152,23 @@ def _reset_cycle(self):
 
     def entity_name(self, name, table=CHARS_REPLACE_TABLE):
         """Format AMQP queue name into a legal SQS queue name."""
-        return text_t(safe_str(name)).translate(table)
```

Comment:
Is that a necessary change in the scope of removing boto2? I am pointing this out because the changes are already very extensive. I think it would be advisable to keep the diff to a minimum.